tests/components/discovergy/test_config_flow.py

```diff
@@ -56,16 +55,21 @@ async def test_reauth(hass: HomeAssistant, mock_meters) -> None:
     assert init_result["type"] == data_entry_flow.FlowResultType.FORM
```

Comment:
Instead of setting up the whole integration above we could also just add a `MockConfigEntry` to Home Assistant.

https://github.com/home-assistant/core/blob/df138d91e69bcac8e6e2860bbb136e94a5efabe0/tests/common.py#L852